### PR TITLE
Moved API functions to backend

### PR DESCRIPTION
**Pull request by a-nahaitsev**
Ask me for the new .env file

### CODE REVIEW

File: electron-src/helpers/API/trelloApi.ts
Comment (d-mashchenko) [inline electron-src/helpers/API/trelloApi.ts:1]: Just as suggestion - can we move on a root level folder "API". Why do we need hide it in helpers?

File: electron-src/index.ts [hunk 1]
```diff
@@ -439,6 +444,37 @@ ipcMain.handle(
...
+    return await getMember(accessToken, options);
+  }
+);
+
+ipcMain.handle("trello:get-cards", async (event, accessToken: string) => {
+  const options = getTrelloOptions();
+
+  return await getCardsOfMember(accessToken, options);
+});
+
```
Comment (d-mashchenko) [inline electron-src/index.ts:477]: Again as suggestion - can we move this code to separate trello api file to make our main a little bit less?

Comment (a-nahaitsev) [reply electron-src/index.ts:477]: idk how it'll implement correctly, we discussed it with you

File: electron-src/helpers/API/trelloApi.ts
Comment (a-nahaitsev) [reply electron-src/helpers/API/trelloApi.ts:1]: when i moved into "helpers" folder's parent, it doesn't work on production, idk why

**Pull request merged**